_partials/nbextensions.md

```diff
@@ -0,0 +1,77 @@
+## `jupyter` notebook extensions
+
+Pimp your `jupyter` notebooks with awesome extensions:
+
+```bash
+# install nbextensions
+pip install jupyter_contrib_nbextensions
+pip install jupyter_nbextensions_configurator
+jupyter contrib nbextension install --user
+jupyter nbextension enable toc2/main
+jupyter nbextension enable collapsible_headings/main
+jupyter nbextension enable spellchecker/main
+jupyter nbextension enable code_prettify/
+jupyter nbextension enable autopep8
```

Comment:
autopep8 and code_prettify seems redundant. Why install both?

Reply:
I'll remove `code_prettify`